### ISSUE TITLE
NTR: D-alanine import across plasma membrane

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#26844]
Please provide as much information as you can: 

* **Suggested term label:**

D-alanine import across plasma membrane

* **Definition (free text)**
The directed import of D-alanine from the extracellular region across the plasma membrane and into the cytosol.

* **Reference, in format PMID:#######**
22418438
25425233

* **Gene product name and ID to be annotated to this term**

Q8ZPA3
Q8ZPA2 
Q8ZPA1
Q8ZPA0

* **Parent term(s)**
GO:0089718
GO:0042941

**Comment by pgaudet** [comment]
@edwong57 
We already have  GO:0042941    D-alanine transport; can you check if that can be renamed 'D-alanine transmembrane transport'; or even 'D-alanine import across plasma membrane'? 
It seems like all annotated genes are bacterial, so that would be the only possible transmembrane process, right? 

Thanks, Pascale

**Comment by pgaudet** [comment]
@edwong57 
Can you please also look at the other siblings: 
[GO:0070777 D-aspartate transport](https://amigo.geneontology.org/amigo/term/GO:0070777#display-lineage-tab)
[GO:0048473 D-methionine transport](https://amigo.geneontology.org/amigo/term/GO:0048473#display-lineage-tab)
[GO:0042942 D-serine transport](https://amigo.geneontology.org/amigo/term/GO:0042942#display-lineage-tab)

and see if the same can be applied (ie move to either 'x import across plasma membrane', or at least 'transmembrane transport? 

Thanks, Pascale

**Comment by deustp01** [comment]
> We already have GO:0042941 D-alanine transport; can you check if that can be renamed 'D-alanine transmembrane transport'; or even 'D-alanine import across plasma membrane'?
> It seems like all annotated genes are bacterial, so that would be the only possible transmembrane process, right?

Import would be the only known process, but as a general strategy, is there any reason to build a direction into a transport term a priori rather than leave it to be specified in the context of a particular annotation? Same quibble about "plasma membrane" rather than just "membrane".

**Comment by pgaudet** [comment]
>is there any reason to build a direction into a transport term a priori rather than leave it to be specified in the context of a particular annotation?


We can discuss this... the issue is that the BP transmembrane transport is no different from the MF transmembrane transporter activity. 

@thomaspd has been arguing that transport should not be a BP, and is always part of a 'larger' process. But for now that's too much refactoring, given all other projects we have open.

**Comment by deustp01** [comment]
> @thomaspd has been arguing that transport should not be a BP, and is always part of a 'larger' process. But for now that's too much refactoring, given all other projects we have open.

Perhaps leaving these transmembrane transports as one-step BPs for now would be a good way to resolve the current issue, and meanwhile the one-step nature of the process would be a useful flag / reminder when the time comes for the systematic refactoring that we want to embed the transport steps into larger processes?

**Comment by edwong57** [comment]
@pgaudet, I checked:

GO:0042941 D-alanine transport,
[GO:0070777 D-aspartate transport](https://amigo.geneontology.org/amigo/term/GO:0070777#display-lineage-tab)
[GO:0048473 D-methionine transport](https://amigo.geneontology.org/amigo/term/GO:0048473#display-lineage-tab)
[GO:0042942 D-serine transport](https://amigo.geneontology.org/amigo/term/GO:0042942#display-lineage-tab)

Not all of the proteins annotated directly to these terms are bacterial, but I checked the one PMID that was associated with a human annotation (SLC1A1) and it looks like a membrane protein, and the majority of the Panther families I checked had plasma membrane as the GO-slim cellular component.

I think it is fine to just add 'transmembrane transport' and add this NTR underneath D-alanine transmembrane transport (GO:0042941 D-alanine transport). D-aspartate already has the same child term.
